src/core/services/aria/aria.js

```diff
@@ -30,18 +30,15 @@ angular
  */
 function MdAriaProvider() {
 
-  var self = this;
-
-  /**
-   * Whether we should show ARIA warnings in the console, if labels are missing on the element
-   * By default the warnings are enabled
-   */
-  self.showWarnings = true;
+  var config = {
```

Comment:
The changes won't be noticeable from outside, so testing this would be unnecessary. Also we still didn't confirm that Angular instantiates the provider with a new instance or by just calling the function.